docs/user-guide/aroma/autoboot.md

```diff
@@ -15,6 +16,22 @@ Currently, every time you want to launch Aroma you will have to launch the Healt
 1. You will be asked if you want to switch the boot title. Press A to select `Switch to PayloadLoader`.
 1. When the process has finished, press A to shutdown the console.
 1. The PayloadLoader will now be launched automatically on every boot.
-    - To change the default boot title hold Start (+) on the Gamepad while booting the console.
+
+
+Now, we are going to make the Aroma environment start automatically when your console loads the Health and Safety Information app and select Wii U Menu as the default boot option.
```

Comment:
The only issue left with the PR is this still isn't under the title tag.

Reply:
For some reason the file on my PC...reset? Tf github

Reply:
Okay done

Reply:
such a github moment